File: uva11461.cpp
#include<stdio.h>
#include<math.h>
int main()
{
    long a,b,i,t,c;
    double r;
    while(scanf("%ld%ld",&a,&b))
    {
        c=0;
        if(a==0&&b==0)return 0;
        for(i=a;i<=b;i++)
        {
            r=sqrt(i);
            t=(int)r;
            if(t==r)c++;
        }
        printf("%ld\n",c);
    }
    return 0;
}

File: UVa easy problems solved in Java/uva_11461.java
import java.util.Scanner;
public class Main
{

    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        //String s;
 
    int a,b,d;
    double c;
    while( sc.hasNext() )
    {
        a = sc.nextInt();
        b = sc.nextInt();
        if(a == 0 && b == 0) break;

        d=0;

        for(; a<=b; a++)
        {
            c=Math.sqrt(a);
            if (a ==(int)c*(int)c)
                ++d;
        }
        System.out.println(d);
    }

    }
}
